styles/mixins/_layout.scss

```diff
@@ -107,3 +91,9 @@ $-color-overlay: var.$color-overlay;
   right: 0;
   background-color: $-color-overlay;
 }
+
+// A content block, with default padding and vertical spacing between elements
```

Comment:
I extracted this out of the `frame` molecule, just kind of adjusting things as it feels natural, but it may not stick around, we'll see.